### PR TITLE
Add support for HTTP/2 in curl

### PR DESCRIPTION
**Pull request by Tomo59**
## Description
Add support for HTTP/2 protocol in curl

## Additional information

Works properly:
- [x] x86_64

I didn't put the binary versions as I can't push to bintray. AlsoI removed the sha256 of curl binaries to remember to change them when curl is recompiled.

### CODE REVIEW

**Comment by uberhacker** [comment]
Thank you for the update @Tomo59!  There is a newer version of curl available now at https://curl.haxx.se/download/curl-7.64.1.tar.xz.  Please try this version and remove the entire binary section when you get a chance.

**Comment by Tomo59** [comment]
OK, I just did that @uberhacker 

version 7.64.1 is working fine for me

**Comment by JL2210** [comment]
Please create pull requests from a branch other than `master` in the future. It allows for making more than one pull request.

File: packages/curl.rb [hunk 1]
```diff
@@ -3,26 +3,27 @@
...
   description 'Command line tool and library for transferring data with URLs.'
   homepage 'https://curl.haxx.se/'
-  version '7.64.0'
-  source_url 'https://curl.haxx.se/download/curl-7.64.0.tar.xz'
-  source_sha256 '2f2f13fa34d44aa29cb444077ad7dc4dc6d189584ad552e0aaeb06e608af6001'
+  version '7.64.1'
+  source_url 'https://curl.haxx.se/download/curl-7.64.1.tar.xz'
+  source_sha256 '9252332a7f871ce37bfa7f78bdd0a0e3924d8187cc27cb57c76c9474a7168fb3'
 
   binary_url ({
```
Comment (JL2210) [inline packages/curl.rb]: Remove the `binary_url` and `binary_sha256` blocks.

**Pull request merged**